react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -421,6 +421,46 @@ Axes.args = {
     },
 };
 
+// Example using "Map" layer. AUses float32 mesh and properties binary arrays. Not PNG.
```

Comment:
Can we move this to a `DeckGLMap/layers/map/mapLayer.story.tsx` file?

Reply:
Done